### DISCUSSION TITLE
Ability to ignore special characters in sorts

### DISCUSSION THREAD

**Discussion by TheHYPO35** [navidrome/navidrome#2958] [Ideas]
I would like to suggest the ability to include special characters at the start of fields from being sorted first ahead of "A" and numbers (e.g. "Weird" Al Yankovic sorts first on my list because of the quotes). Songs that start with parentheses (e.g.: (I've Had) The Time of My Life) sort first. The IgnoredArticles field doesn't seem to affect special characters like parentheses and including quotes or apostrophes seem to break the config file even with an escaping backslash.

**Comment by deluan** [comment]
This can be done by using the "Sort*" tags for customizing your artists, albums and tracks, and then setting `PreferSortTags` to true.

**Reply by TheHYPO35** [reply]
That’s true, but that requires manual intervention and re-tagging a collection. It also requires regularly monitoring your collection for new offending tags. 

An automated solution that includes these characters that would commonly be  desired to be ignored would be perpetual and preferable, And doesn’t seem like it would be terribly difficult to implement. Either with an escape character, or a stand-in character in the config file.

**Reply by deluan** [reply]
Yeah, I agree. But I don't want to add this to the current scanner. I'll add this option to the new scanner #2709.
